src/tools/kvstore_tool.cc

```diff
@@ -55,6 +55,8 @@ StoreTool::StoreTool(const string& type,
   }
 }
 
+#ifdef WITH_BLUESTORE
```

Comment:
Shouldn't we wrap load_bluestore() declaration in kvstore_tool.h with the same ifdef as well then?

Reply:
We could do that, it's more correct, but it's not for free: it adds more preprocessor clutter to the header. I can amend the commit, do you want that?

I was wondering why this method is declared `public`. It's only used internally and having it public makes no sense.

Reply:
>We could do that, it's more correct, but it's not for free: it adds more preprocessor clutter to the header. 
That's why I proposed to put ifdef inside load_bluestore only in my initial comment ;)

>I can amend the commit, do you want that?
better do IMO.

>I was wondering why this method is declared public. It's only used internally and having it public makes no sense.

Honestly, no idea. Perhaps no one just didn't care...

Reply:
Done - added ifdef to header and made the method private.